packages/next/src/server/lib/incremental-cache/index.ts

```diff
@@ -58,7 +58,7 @@ export class CacheHandler {
     ..._args: Parameters<IncrementalCache['set']>
   ): Promise<void> {}
 
-  public async revalidateTag(_tag: string): Promise<void> {}
+  public async revalidateTag(_tag: string | string[]): Promise<void> {}
```

Comment:
should we update the `file-system-cache` API as well since it also implements `CacheHandler`?